novuhq/website
# feat: add digest page

**Describe what changes this pull request brings**

Add Digest page

**Steps to check:**

1. Open [preview](https://deploy-preview-302--novu-website.netlify.app/digest/)
2. Make sure that everything looks and works as expected

**BEFORE MERGE:**

- [ ] Add appropriate links to all sections
